Jenkinsfile

```diff
@@ -171,6 +171,7 @@ pipeline {
                 env.SYSTEM_ASSIGNED_IDENTITY = sh(script: 'ci/authn-azure/get_system_assigned_identity.sh', returnStdout: true).trim()
 
                 sh('summon -f ci/authn-azure/secrets.yml ci/test cucumber_authenticators_azure')
+                stash name: 'testResultAzure', includes: "cucumber/*/*.*,container_logs/*/*,spec/reports/*.xml,spec/reports-audit/*.xml,cucumber/*/features/reports/**/*.xml"
```

Comment:
thanks @eladkug ! do we need this also for authn-gcp tests?

Reply:
Nope. I verified we have gcp as well, and added it to our html reports too.
![image](https://user-images.githubusercontent.com/59792097/97849739-aa08bb00-1cfb-11eb-8053-058e039c0460.png)

Reply:
ohhh do we need to have this explicitly for azure because it runs in an azure VM and not in the Jenkins node? this can explain how it vanished because we moved the azure tests to run inside the azure VM a while ago.

if this is correct, can you please add a comment saying that?
something like `We need to stash the test results so they are available on the Jenkins node`